syntax="proto3";
package IM;

message LoadFriendInfo
{
	int32 networkType=1;
	int32 msgID=2;//每条消息的标识
	int32 recvID=3;//消息接收者
	int32 sendId=4;
	int32 userID=5;
	int32 friendID=6;
	bytes userRemarkName=7;
	int32 state=8;
	bytes f_group=9;
	int32 u_sex=10;
	int32 f_sex=11;
	bytes friendRemarkName=12;
}
